src/Compilers/Shared/ShadowCopyAnalyzerAssemblyLoader.cs

```diff
@@ -9,8 +9,8 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-// The ShadowCopyAnalyzerAssemblyLoader derives from DesktopAnalyzerAssemblyLoader (NET472) OR CoreClrAnalyzerAssemblyLoader (NETCOREAPP3_1)
-#if NET472 || NETCOREAPP3_1
+// The ShadowCopyAnalyzerAssemblyLoader derives from DesktopAnalyzerAssemblyLoader (NET472) OR CoreClrAnalyzerAssemblyLoader (NETCOREAPP)
+#if NET472 || NETCOREAPP
```

Comment:
Is there a similar change we should be making around the NET472 stuff?

Reply:
@jasonmalinowski 

Here was my logic. The expectation is that we're going to be moving .NET Core forward at a regular cadence. Hence moving to `NETCOREAPP` has value because it reduces churn in future updates.

There is no expectation that we'll ever move beyond `NET472`. In case we did it would be to move to `NET48`. At that point we're done because there are no future desktop versions. Hence moving to a generic desktop directive is at best a net even result (and in the likely case unneeded work)

Reply:
Ah, right, didn't think about the "if we change NET472 we're only changing once period" rule. Sounds good.